.travis.yml

```diff
@@ -39,7 +39,7 @@ script:
     then
       bash scripts/test-make-requirements.sh
     fi
-  - "scripts/docker test --noinput --stop --verbosity=2 --divide-depth=1 --with-timing --threshold=10"
+  - "scripts/docker test --noinput --verbosity=2 --divide-depth=1 --with-timing --threshold=10"
```

Comment:
Will have a negative impact on queue throughput. Is this just a temporary change to find slow tests (to be reverted later before this PR is merged), or are you intending to keep it indefinitely?

Reply:
yep this is just temporary and it doesn't work that well anywyas to some test cleanup issues i think.

I'm probably going to lower the threshold for logging to get more logs at once and give these tests some more buffer. They appear to be more variable that i thought